R/style_guides.R

```diff
@@ -49,7 +50,8 @@ NULL
 tidyverse_style <- function(scope = "tokens",
                             strict = TRUE,
                             indent_by = 2,
-                            start_comments_with_one_space = FALSE) {
+                            start_comments_with_one_space = FALSE,
+                            reindention = specify_reindention()) {
```

Comment:
`default_reindention()`? What is the rationale behind the `specify_` prefix?

Reply:
We can use a forwarder here too to be consistent with  #221.

Reply:
Same line of argument as in https://github.com/krlmlr/styler/pull/221#discussion_r142003169. I suggest to revert to `specify_reindention()`.

Reply:
@krlmlr ok?